broker/api.py

```diff
@@ -386,3 +383,11 @@ def parse_header_options(params):
 def normalize_header_list(headers):
     headers = {header.upper() for header in headers}
     return sorted(list(headers))
+
+
+def parse_domain_options(params):
+    domains = params.get("domains", None)
+    if isinstance(domains, str):
+        return [d.strip().lower() for d in domains.split(",")]
+    if isinstance(domains, list):
+        return domains
```

Comment:
Should we clean the strings when it's a list too?

Reply:
that would probably make sense.

Reply:
fixed in bc98c399731e720b1e94faec7a7cbc3f80bcf012